### PR TITLE
docs(environment-variables): make EXECUTIONS_TIMEOUT and EXECUTIONS_T…

### PR DESCRIPTION
**Pull request by ultrabug**
…IMEOUT_MAX more explicit

EXECUTIONS_TIMEOUT and EXECUTIONS_TIMEOUT_MAX environment variables have a relationship that was not obvious on their documentation

### CODE REVIEW

File: docs/hosting/environment-variables/environment-variables.md [hunk 1]
```diff
@@ -143,8 +143,8 @@ Refer to [User management](/hosting/authentication/user-management-self-hosted/)
 | :------- | :---- | :------- | :---------- |
 | `EXECUTIONS_PROCESS` | Enum string: `main`, `own` | `own` | Whether n8n executions run in their own process or the main process. <br><br>Refer to [Execution modes and processes](/hosting/scaling/execution-modes-processes/) for more details. |
 | `EXECUTIONS_MODE` | Enum string: `regular`, `queue` | `regular` | Whether executions should run directly or using queue.<br><br>Refer to [Execution modes and processes](/hosting/scaling/execution-modes-processes/) for more details. |
-| `EXECUTIONS_TIMEOUT` | Number | `-1` | The maximum run time (in seconds) before stopping a workflow execution. Set to `-1` to disable. |
-| `EXECUTIONS_TIMEOUT_MAX` | Number | `3600` | The maximum execution time (in seconds) for an individual workflow. |
+| `EXECUTIONS_TIMEOUT` | Number | `-1` | Whether executions should be stopped after this maximum execution time (in seconds). Can be overriden per workflow up to `EXECUTIONS_TIMEOUT_MAX`. Set to `-1` to disable. |
```
Comment (StarfallProjects) [inline docs/hosting/environment-variables/environment-variables.md:146]:
```suggestion
| `EXECUTIONS_TIMEOUT` | Number | `-1` | Set an execution time (in seconds), after which n8n stops the execution. Users can override this for individual workflows up to the duration set in `EXECUTIONS_TIMEOUT_MAX`. Set `EXECUTIONS_TIMEOUT` to `-1` to disable. |
```

File: docs/hosting/environment-variables/environment-variables.md [hunk 2]
```diff
@@ -143,8 +143,8 @@ Refer to [User management](/hosting/authentication/user-management-self-hosted/)
...
+| `EXECUTIONS_TIMEOUT_MAX` | Number | `3600` | The maximum execution time (in seconds) that can be configured for an individual workflow. |
```
Comment (StarfallProjects) [inline docs/hosting/environment-variables/environment-variables.md:147]:
```suggestion
| `EXECUTIONS_TIMEOUT_MAX` | Number | `3600` | The maximum execution time (in seconds) that users can set for an individual workflow. |
```

**Comment by StarfallProjects** [comment]
Hi @ultrabug thank you for the contribution. I've suggested two small rephrasings.

File: docs/hosting/environment-variables/environment-variables.md (hunk 1 above)
Comment (StarfallProjects) [reply docs/hosting/environment-variables/environment-variables.md:146]: Sounds good

Comment (ultrabug) [reply docs/hosting/environment-variables/environment-variables.md]: done

**Comment by StarfallProjects** [comment]
🎉

**Pull request merged**